src/main/groovy/au/org/ala/merit/reports/NHTOutputReportData.groovy

```diff
@@ -14,6 +15,15 @@ class NHTOutputReportData extends ReportData {
     ProjectService projectService
 
     Map getContextData(Map project) {
+        // Side effect - filter data sets.
+        project.custom.dataSets = new JSONArray(project.custom.dataSets?.findAll {
+            it.progress == ActivityService.PROGRESS_FINISHED
```

Comment:
is this meant to also do filtering on progress?

Reply:
Yes, we only want users to be able to select "finished" data sets anyway, so this is just so we are sending less data since we are pre-processing anyway.  I'm probably going to add a filter for "not submitted or approved" as well but just have to make sure the pre-pop is turned off in view mode first or it will probably stuff up the view mode version of the data set.

Reply:
> I'm probably going to add a filter for "not submitted or approved" as well but just have to make sure the pre-pop is turned off in view mode first or it will probably stuff up the view mode version of the data set.

Actually, this is wrong - the published data sets still need to be selectable in the "Synthesising/finalising data sets service" so likely we'll leave it as is

Reply:
Sorry, I misinterpreted your comment -you are right that filtering is broken - that's been fixed now in the commit below.